src/hooks/useNetInfo.ts

```diff
@@ -2,7 +2,7 @@ import NetInfo from '@react-native-community/netinfo';
 import {useEffect, useState} from 'react';
 
 export function useNetInfo() {
-  const [isConnected, setIsConnected] = useState(false);
+  const [isConnected, setIsConnected] = useState(true);
```

Comment:
Why would we want to set the initial state to`true`. In any case the initial state should be `false`. Is this fix related to the current scope of the bug?